client/src/components/icons/FavoriteIcon.tsx

```diff
@@ -0,0 +1,56 @@
+import {StyleSheet, TouchableOpacity, View} from "react-native";
+import {MaterialCommunityIcons} from "@expo/vector-icons";
+import {updateFavorites} from "../../services/favorites.servise";
+import {useAppDispatch, useAppSelector} from "../../store/hooks";
+import {selectFavorites} from "../../store/favorites/favorites.selectors";
+import {addFavoriteThunk} from "../../store/favorites/favorites.slice";
+
+interface Props {
+    itemId: string
+}
+
+export const IconsStyles = StyleSheet.create({
+    iconCircleWrapper: {
+        position: "relative",
+        width: 32,
+        height: 32,
+        borderRadius: 50,
+        overflow: "hidden",
+    },
+    iconCirclePaper: {
+        position: "absolute",
+        top: "-35%",
+        left: "-35%",
+        width: "170%",
+        height: "170%",
+        backgroundColor: "white",
```

Comment:
The white backgroundColor doesn't look good when using TTS icon in the popup, where it should be transparent.
I'd remove this property from here and add it as props to the component, which will be added to styles in `<View style={IconsStyles.iconCirclePaper}/>`. The default should be one of `transparent` or `white` (define the most used color as the default).

Reply:
done